cloudformation/tests/test_template_syntax.py

```diff
@@ -0,0 +1,15 @@
+import json
+
+
+def test_scheduled_events_template_syntax():
```

Comment:
Thanks for translating this as a unit test. we need to enable `cfn-tests` in travis as part of `CloudFormation Templates Checks`